projects/projects.md

```diff
@@ -1,18 +1,22 @@
 ---
-title: "Projects"
-author: "Benedikt Hegner"
+title: "HSF Affiliate Projects"
```

Comment:
Not "Affiliated"? Same comment just below.

Reply:
Yes, should be affiliated!